exercises/nucleotide-count/canonical-data.json

```diff
@@ -28,6 +28,17 @@
           }
         },
         {
+          "description": "can count one nucleotide in single-character input.",
```

Comment:
presently the only description that ends in punctuation, until other descriptions are changed. This means that inevitably this will cause to discuss:

* whether the descriptions in this file should, in general, end in punctuation
* is the (temporary, right?) inconsistency OK

Reply:
My vote is for consistency.  Punctuation should be a separate PR, or at the very least a separate commit.